src/static/css/2019.css

```diff
@@ -49,6 +49,25 @@ select {
   color: white;
 }
 
+a.navigation-logo {
+  outline: none;
+  color: white;
+}
+a.navigation-logo img:hover,
+a.navigation-logo:focus img {
+  filter: brightness(0) saturate(100%) invert(86%) sepia(4%) saturate(4357%) hue-rotate(16deg) brightness(136%) contrast(94%);
+}
+
+header.alt-bg a:hover, footer.alt-bg a:hover,
+header.alt-bg a:focus, footer.alt-bg a:focus {
+  color: #F7F779;
+  outline: none;
+}
+header.alt-bg a.interactive:hover, footer.alt-bg a.interactive:hover,
```

Comment:
I think this can be simplified to `a:hover, a:focus { text-decoration: underline; }`

Are there any cases where we don't want a link to have an underline?

Reply:
I agreed :+1: 
Yes, I working on it now.